src/main/scala/gemmini/Configs.scala

```diff
@@ -35,60 +35,42 @@ class WithMultiRoCC extends Config((site, here, up) => {
 // -----------------------
 
 object GemminiConfigs {
-  // import Arithmetic.FloatArithmetic._
-
   val defaultConfig = GemminiArrayConfig[SInt, Float, Float](
-  // val defaultConfig = GemminiArrayConfig[Float, Float](
     opcodes = OpcodeSet.custom3,
+
     tileRows = 1,
     tileColumns = 1,
-    // meshRows = 4,
-    // meshColumns = 4,
     meshRows = 16,
     meshColumns = 16,
+
     ld_queue_length = 8,
     st_queue_length = 2,
     ex_queue_length = 8,
+
     rob_entries = 16,
+    hasIm2col = true, //declare im2col block
```

Comment:
The im2col block is broken right now (it can't handle single-ported SRAMs). We should set this to `false` until it's fixed.

Reply:
We can set this to false for now

Reply:
Conv-fsm hasn't fixed this; it doesn't use the im2col module currently.